hash/sega_beena_cart.xml

```diff
@@ -256,24 +508,26 @@ license:CC0
 		<description>Samurai Sentai Shinkenger Battle ga Ippai! Iza Mairu!</description>
 		<year>2009</year>
 		<publisher>Bandai</publisher>
+		<info name="alt_title" value="侍戦隊シンケンジャー バトルがいっぱい!いざ参る!"/>
```

Comment:
This also has halfwidth exclamation marks.

Reply:
Replaced all halfwidth with regular chars and removed spaces!

Reply:
No, that commit replaced fullwidth characters with halfwidth characters.  You’re supposed to use fullwidth punctuation in Japanese text, not halfwidth.